show/main.py

```diff
@@ -216,6 +216,37 @@ def get_interface_mode():
     return mode
 
 
+#
+# Use this method to validate unicast IPv4 address
+#
+def is_ip4_addr_valid(addr, display):
```

Comment:
As suggested, please use utility functions already available from utilities_common/cli.py. If its new, kindly add this to utilities so that it can be used by other components.

Reply:
Done.